website/docs/r/mariadb_server.html.markdown

```diff
@@ -0,0 +1,103 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_mariadb_server"
+sidebar_current: "docs-azurerm-resource-database-mariadb-server"
+description: |-
+  Manages a MariaDB Server.
+---
+
+# azurerm_mariadb_server
+
+Manage a MariaDB Server.
+
```

Comment:
given this resource is in Preview can we add a disclaimer to it? e.g. https://www.terraform.io/docs/providers/azurerm/r/shared_image_gallery.html

Reply:
Good point... added.